src/content-layout/interfaces.ts

```diff
@@ -1,6 +1,6 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-import { BaseComponentProps } from '../../base-component';
```

Comment:
@FlorianDr Wasn't there something here that you indicated needed to be changed?

Reply:
Yes, we were missing the usually plumbing with `getBaseProps`.